### PR TITLE
Add STATS types for online opers, ports and server uptime

### PR DESCRIPTION
**Pull request by Heufneutje**
Pretty much what it says on the tin. I also added onlineopers and uptime to the public_info node in the example config. I think that was one of the things that came up when we initially wrote the config for 0.3.

### CODE REVIEW

File: conf/extra-modules-example.yaml [hunk 1]
```diff
@@ -148,6 +148,18 @@ modules:
 # notifies opers of user disconnections on remote servers.
 #- ServerNoticeRemoteQuit
 
+# StatsOnlineOpers: Provides the ONLINEOPERS STATS type, which lists all opers
```
Comment (ElementalAlchemist) [inline conf/extra-modules-example.yaml]: The STATS types are implemented in lowercase, so we should have them that here as well.

File: txircd-example.yaml [hunk 2]
```diff
@@ -177,8 +177,12 @@
 # This controls which STATS options are available to everyone rather than just
 # opers. Options not listed here will be available only to opers.
 # Specify this as a list of options. If not specified, the default is no
-# options.
-#public_info: []
+# options. The example shown below requires the StatsOnlineOpers and
+# StatsUptime modules to be loaded; see the extra modules include at the bottom
+# of this file for more.
```
Comment (ElementalAlchemist) [inline txircd-example.yaml]: It technically doesn't require these modules. If they're not loaded, this just won't do anything.

File: txircd/modules/extra/stats_onlineopers.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+	name = "StatsOnlineOpers"
+
+	def actions(self):
+		return [ ("statsruntype-onlineopers", 10, self.listOnlineOpers) ]
+
+	def listOnlineOpers(self):
+		info = {}
+		for user in self.ircd.users.itervalues():
+			if "o" in user.modes:
```
Comment (ElementalAlchemist) [inline txircd/modules/extra/stats_onlineopers.py]: The correct way to check this is with the userhasoperpermission action.

File: conf/extra-modules-example.yaml (hunk 1 above)
Comment (Heufneutje) [inline conf/extra-modules-example.yaml]: We should update the oper documentation as well then. Cause they're all in lowercase there.

File: txircd-example.yaml (hunk 2 above)
Comment (Heufneutje) [inline txircd-example.yaml]: I specified it because we did so in other places (see channel_exempt_level)

**Comment by Heufneutje** [comment]
This should be ready now.

**Pull request merged**